overrides/groovy/postInit/Post-Initial/Main/Mod-Specific/jei.groovy

```diff
@@ -4,6 +4,64 @@ import com.nomiceu.nomilabs.util.LabsModeHelper
 
 // AR
 mods.hei.removeAndHide(item('advancedrocketry:crystal:*')) // Random Crystal Blocks
+mods.hei.removeAndHide(item('densemetals:dense_platinum_ore')) //Dense Platinum ore
```

Comment:
Capatilize all words of all comments. For entries where you have the metadata, seperate that from the main string. Seperate GregTech and Dense Ores. Also, please use the `metaitem` bracket handler instead of `item` where possible.

Also check if the ores are obtainable in HM/NM only, a mode specific check may be needed there.